src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -1,70 +1,118 @@
-                <SerifSemibold18 mb={4}>You're the highest bidder</SerifSemibold18>
-
-                <Sans12 color="black60">Time left</Sans12>
-                <Timer timeLeftInMilliseconds={1000 * 60 * 20} />
-              </TopOffset>
+              <CenteredView>
+                <Icon20 source={require("../../../../../images/circle-check-green.png")} />
+                <Title m={4}>You're the highest bidder</Title>
+                <TimeLeftToBidDisplay timeLeftToBid={timeLeftToBid} timeLeftToBidMillis={timeLeftToBidMillis} />
+              </CenteredView>
             </View>
-          </CenteringContainer>
+            <BidGhostButton text="Continue" onPress={() => null} />
+          </Container>
         </BiddingThemeProvider>
       )
     } else {
+      const bidAgain = SHOW_TIMER_STATUSES.indexOf(this.props.status) > -1
+      const buttonMsg = bidAgain ? `Bid ${this.props.sale_artwork.current_bid.display} or more` : "Continue"
       return (
         <BiddingThemeProvider>
-          <Container>
-            <CenteringContainer>
-              <Icon20 m={2} source={require("../../../../../images/circle-x-red.png")} />
-              <SerifSemibold18 mb={4}>{this.props.message_header}</SerifSemibold18>
-              {/* TODO: parse makedown and render links */}
-              <StyledText maxWidth={280}>{this.props.message_description_md}</StyledText>
-
-              <Sans12 color="black60">Time left</Sans12>
-              <Timer timeLeftInMilliseconds={1000 * 60 * 20} />
-
-              <Divider mt={5} mb={4} />
-
-              <Serif16>Current bid</Serif16>
-              <SerifSemibold18>$45,000</SerifSemibold18>
-            </CenteringContainer>
-
-            <Button text="Bid again" onPress={() => null} />
+          <Container mt={40}>
+            <View>
+              <CenteredView>
+                <Icon20 source={require("../../../../../images/circle-x-red.png")} />
+                <Title m={4}>{this.props.message_header}</Title>
+                <MarkdownRenderer>{this.props.message_description_md}</MarkdownRenderer>
+                {bidAgain && (
+                  <TimeLeftToBidDisplay timeLeftToBid={timeLeftToBid} timeLeftToBidMillis={timeLeftToBidMillis} />
+                )}
+              </CenteredView>
+            </View>
+            {bidAgain ? (
+              <Button
+                text={buttonMsg}
+                onPress={() => {
+                  this.onPressBidAgain()
+                }}
+              />
+            ) : (
+              <BidGhostButton text="Continue" onPress={() => null} />
+            )}
           </Container>
         </BiddingThemeProvider>
       )
     }
   }
 }
 
-const TopOffset = styled.View`
-  top: -50%;
+const TimeLeftToBidDisplay = props => {
+  return (
+    <CenteredView>
+      <Sans12>Ends {moment(props.timeLeftToBid, "YYYY-MM-DDTHH:mm:ss+-HH:mm").format("MMM D, ha")}</Sans12>
```

Comment:
Given that we always show the date and hour I think it makes more sense to move this into the `<Timer>` component.